ansible/roles/pyxis/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Clone pyxis
```

Comment:
can you duplicate the pattern above by testing whether `/usr/local/lib/slurm/spank_pyxis.so` exists as the first task and then have these 3 tasks be in a block that only executes when it does not?

Reply:
Done.